README.md

```diff
@@ -1,2 +1,22 @@
 # VirtualDog
 An easy to grasp example for training on various web technologies
+
+```
+npm install
+npm run bower
+npm run typings
+npm start
+```
+
+Requires the following global installations:
+
+- npm
+- bower
```

Comment:
If you are using npm run for bower and others you actually don't need a global install of them since npm knows how to call them locally after npm install.